templates/contribute/style.md

```diff
@@ -282,184 +278,221 @@ example (n : ℝ) (h : 1 < n) : 0 < n := by linarith
 is preferred over
 
 ```lean
-example (n : ℝ) : 1 < n → 0 < n := λ h, by linarith
+example (n : ℝ) : 1 < n → 0 < n := fun h ↦ by linarith
 ```
 
 and
 
 ```lean
-example (n : ℕ) : 0 ≤ n := dec_trivial
+example (n : ℕ) : 0 ≤ n := Nat.zero_le n
 ```
 
 is preferred over
 
 ```lean
-example : ∀ (n : ℕ), 0 ≤ n := λ n, dec_trivial
+example : ∀ (n : ℕ), 0 ≤ n := Nat.zero_le
 ```
 
 ### Binders
 
 Use a space after binders:
 ```lean
 example : ∀ α : Type, ∀ x : α, ∃ y, y = x :=
-λ (α : Type) (x : α), exists.intro x rfl
+  fun (α : Type) (x : α) ↦ Exists.intro x rfl
 ```
 
+### The anonymous function arrow
+
+Lean has several nice syntax options for declaring anonymous functions. For very simple
+functions, one can use the centered dot as the function argument, as in `(· ^ 2)` to 
+represent the squaring function. However, sometimes it is necessary to refer to the 
+arguments by name (e.g., if they appear in multiple places in the function body). The
+Lean default for this is `fun x => x * x`, but mathlib introduces an alternative 
+notation which we prefer to use in mathlib: `fun x ↦ x * x` (inserted with `\mapsto`).
```

Comment:
This is not introduced in mathlib, it comes from core Lean. The only thing that mathlib does is to use the option to print this notation.

Reply:
Ah thanks, I had that wrong. I think what I said is correct now?